### PR TITLE
Improve the connector for better user experience at Choreo

### PR DESCRIPTION
**Pull request by SanduDS**
Fixes https://github.com/wso2-enterprise/choreo/issues/4865

## Approach
1] Updated display annotations
2] Added new test cases
3] Removed time module return type
4] updated the GitHub actions

## Automation tests
 - Unit tests 
   > Done
 - Integration tests
   > Done

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

**Review by LakshanSS — COMMENTED** [review]
In release.yml, there is one more place (Line no 27) to change the ballerina action from 'master' to 'slalpha5'

**Comment by SanduDS** [comment]
> In release.yml, there is one more place (Line no 27) to change the ballerina action from 'master' to 'slalpha5'

@LakshanSS Thank you, fixed it

**Comment by SanduDS** [comment]
> In release.yml, there is one more place (Line no 27) to change the ballerina action from 'master' to 'slalpha5'
@LakshanSS Thank you, fixed it

File: client.bal [hunk 1]
```diff
@@ -106,7 +105,7 @@ public client class Client {
     #
     # + account - Account type record with detail
     # + return - If success returns a RecordAddResponse type record otherwise the relevant error
-    @display{label: "Add new account"}
+    @display{label: "Add a new account record"}
```
Comment (abeykoon) [inline client.bal:108]: I would say `Add New Account` is better. Shall we update the display annotation accordingly?

File: client.bal [hunk 2]
```diff
@@ -215,7 +214,8 @@ public client class Client {
 
     # Retrieves all currency types instances from NetSuite
     #
-    # + return - If success returns an array of Currencies otherwise the relevant error
+    # + return - If success returns an array of currency records otherwise the relevant error
+    @display{label: "Get all currency type records"}
```
Comment (abeykoon) [inline client.bal:218]: `Get All Currency Types`

File: client.bal (hunk 1 above)
Comment (SanduDS) [reply client.bal]: Resolved. Removed the articles too.

**Review by abeykoon — APPROVED** [review]

**Pull request merged**